sftp/replica_client.go

```diff
@@ -128,6 +127,44 @@ func (c *ReplicaClient) Init(ctx context.Context) (_ *sftp.Client, err error) {
 	return c.sftpClient, nil
 }
 
+func (c *ReplicaClient) getSigner() (ssh.Signer, error) {
+	buf, err := os.ReadFile(c.KeyPath)
+	if err != nil {
+		return nil, fmt.Errorf("cannot read sftp key path: %w", err)
+	}
+
+	signer, err := ssh.ParsePrivateKey(buf)
+	if err != nil {
+		return nil, fmt.Errorf("cannot parse sftp key path: %w", err)
+	}
+
+	return signer, nil
+}
+
+func (c *ReplicaClient) getCertSigner(priv ssh.Signer) (ssh.Signer, error) {
+	certBuf, err := os.ReadFile(c.KeyPath + "-cert.pub")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should there be a separate certificate path (e.g. `c.CertPath`)? That seems better than assuming the user has the cert relative to the key path.

Reply:
Most SSHy things have kind of a convention around the names of public keys and certs relative to the private key. If you `ssh -i ~/foobar`, it'll try both the plan key and `~/foobar-cert.pub`. That said,  there's no harm in being explicit and using a separate config if you'd rather.

Reply:
I think I'd prefer it be explicit if the convention isn't required. Also, I'm rolling back replication changes that ended up moving to LiteFS so I'll merge this in after I finish that.